src/ruby/puppetserver-lib/puppet/server/master.rb

```diff
@@ -47,8 +47,14 @@ def initialize(puppet_config, puppet_server_config)
     @catalog_compiler = Puppet::Server::Compiler.new
 
     if multithreaded
```

Comment:
Are we running multithreaded in CI?

Reply:
yes. I think so? at least we are in travis...

Reply:
I know we are in Travis, I guess I was wondering how we encountered this issue in the Jar Jar tests. but I guess it's probably not this part but the `initialize_settings` part, which isn't inside a `multithreaded` conditional, that caused the error.

Reply:
so I guess I answered my own question lol

Reply:
Not yet, apart from the travis unit tests.